$NetBSD: patch-ak,v 1.4 2012/10/27 17:39:12 wiz Exp $

Add a patch to handle systems which have uint64_t but not the UINT64_C
macro.  This could happen with UNIX98-type systems, such as the code on
our netbsd-1-5 branch, and would prevent this package from building on
such systems.  Problem also reported to the original author.

--- include/types.h.orig	2005-07-27 17:02:56.000000000 +0000
+++ include/types.h
@@ -104,7 +104,12 @@ typedef unsigned long u32;
 #undef u64	    /* maybe there is a macro with this name */
 #if SIZEOF_UINT64_T == 8
 typedef uint64_t u64;
+#ifdef UINT64_C
 #define U64_C(c) (UINT64_C(c))
+#else
+	/* make a best guess, could happen with UNIX98 <inttypes.h> */
+#define U64_C(c) (c)
+#endif
 #define HAVE_U64_TYPEDEF
 #elif SIZEOF_UNSIGNED_INT == 8
 typedef unsigned int u64;
